societe-francaise-degyptologie.csl

```diff
@@ -0,0 +1,212 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="note" version="1.0" delimiter-precedes-et-al="never" delimiter-precedes-last="never" initialize-with="." page-range-format="expanded" demote-non-dropping-particle="never">
+  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Société française d'égyptologie</title>
+    <title-short>SFE</title-short>
+    <id>http://www.zotero.org/styles/societe-francaise-degyptologie</id>
+    <link href="http://www.zotero.org/styles/societe-francaise-degyptologie" rel="self"/>
+    <link href="https://www.sfe-egyptologie.fr/index.php/publications/recommandations-aux-auteurs" rel="documentation"/>
```

Comment:
If you started from an existing style, can you please add a "template" link to it here?

Reply:
Hello, I created the style from scratch. Thanks for your help